app/controllers/reports/subscription_reports_for_project_owners_controller.rb

```diff
@@ -0,0 +1,24 @@
+require 'csv'
+
+class Reports::SubscriptionReportsForProjectOwnersController < Reports::BaseController
+  def index
+    if policy(Project.find(params[:project_id])).update?
+      @report = end_of_association_chain.to_csv
+      respond_to do |format|
```

Comment:
Your action isn't returning anything different than CSV, for this reason the `respond_to` is unnecessary